src/main/scala/it/pagopa/interop/tenantprocess/error/ResponseHandlers.scala

```diff
@@ -144,6 +144,19 @@ object ResponseHandlers extends AkkaResponses {
       case Failure(ex)                                    => internalServerError(ex, logMessage)
     }
 
+  def updateVerifiedAttributeExtensionDateResponse[T](logMessage: String)(
+    success: T => Route
+  )(result: Try[T])(implicit contexts: Seq[(String, String)], logger: LoggerTakingImplicit[ContextFieldsToLog]): Route =
+    result match {
+      case Success(s)                                     => success(s)
+      case Failure(ex: ExpirationDateCannotBeInThePast)   => badRequest(ex, logMessage)
+      case Failure(ex: OrganizationNotFoundInVerifiers)   => forbidden(ex, logMessage)
+      case Failure(ex: VerifiedAttributeNotFoundInTenant) => notFound(ex, logMessage)
+      case Failure(ex: ExtensionDateNotFoundInVerifier)   => notFound(ex, logMessage)
+      case Failure(ex: ExpirationDateNotFoundInVerifier)  => notFound(ex, logMessage)
```

Comment:
hits should probably be a bad request
```suggestion
      case Failure(ex: ExpirationDateNotFoundInVerifier)  => badRequest(ex, logMessage)
```